src/DataFrames.jl

```diff
@@ -91,6 +91,23 @@ else
     export only
 end
 
+if VERSION >= v"1.3"
+    using Base.Threads: @spawn
+else
+    # This is the definition of @async in Base
+    macro spawn(expr)
```

Comment:
the definition of `@spawn` is:
```
macro spawn(expr)
    letargs = Base._lift_one_interp!(expr)

    thunk = esc(:(()->($expr)))
    var = esc(Base.sync_varname)
    quote
        let $(letargs...)
            local task = Task($thunk)
            task.sticky = false
            if $(Expr(:islocal, var))
                put!($var, task)
            end
            schedule(task)
            task
        end
    end
end
```
and I understand we do not need the `letargs` variable, but why the other things are changed?

Reply:
Ah - you mean that on 1.0 we just do @async but without threading?

Reply:
Yes. That's the best we can do there.